docs/README.md

```diff
@@ -85,20 +73,32 @@ Documentation of the standard library.
 
 Discover the ecosystem of Crystal libraries.
 
+* [Writing Shards](guides/writing_shards.md)
```

Comment:
Can we put this as the last entry? I think it's the most advanced topic and shouldn't be the first one.